libs/components/indicators/src/lib/modules/label/label.component.ts

```diff
@@ -24,7 +24,7 @@ export class SkyLabelComponent {
     if (this.#_labelType === undefined) {
       this.labelTypeOrDefault = LABEL_TYPE_DEFAULT;
     } else {
-      this.labelTypeOrDefault = this.#_labelType;
+      this.labelTypeOrDefault = this.labelType;
```

Comment:
Should be `this.#_labelType` so that we don't invoke the getter inside the setter.

Reply:
Better yet:
```
this.labelTypeOrDefault = (this.#_labelType === undefined) 
  ? LABEL_TYPE_DEFAULT
  : this.labelType;
```

Reply:
I tend to only reference `value` and not `this.#_whateverBackingField` because it's ever so slightly more efficient.

Reply:
Changed to reference `value`